mithril-stm/src/multi_sig.rs

```diff
@@ -229,12 +231,12 @@ impl VerificationKeyPoP {
     // two final exponantiations (for verifying k1 and k2) into a single one.
     pub fn check(&self) -> Result<(), MultiSignatureError> {
         use blst::{
-            blst_fp12, blst_fp12_finalverify, blst_p1_affine_generator, blst_p1_to_affine,
-            blst_p2_affine, blst_p2_affine_generator, BLST_ERROR,
+            blst_fp12, blst_fp12_finalverify, blst_p1_affine_generator, blst_p2_affine_generator,
+            BLST_ERROR,
         };
         let result = unsafe {
             let g1_p = *blst_p1_affine_generator();
-            let mvk_p = std::mem::transmute::<&BlstPk, &blst_p2_affine>(&self.vk.0);
+            let mvk_p = std::mem::transmute::<&BlstVk, &blst_p2_affine>(&self.vk.0);
```

Comment:
We need to have *transmute-free* `multi-sig`, after transmute PR is done, we should change all occurrences in this PR as well.

Reply:
Yes. My suggestion is that we merge this, and then we work on removing transumute calls from another PR.